colossalai/fx/profiler/dataflow.py

```diff
@@ -69,6 +73,7 @@ def is_phase(n: Node, phase: Phase) -> bool:
     return n.meta['phase'] == phase
 
 
+@compatibility(is_backward_compatible=False)
```

Comment:
`autograd_graph_analysis` is only available when `MetaTensor` is available to trace the graph of backward.